pyproject.toml

```diff
@@ -115,3 +115,12 @@ max-doc-length = 79
 
 [tool.ruff.pydocstyle]
 convention = "numpy"
+
+[tool.coverage.report]
+exclude_lines = [
+    "pragma: no cover",
+    "raise AssertionError",
```

Comment:
I can understand the others, but why wouldn't you want coverage of AsssertionErrors, or allow lines to be declared untestable by fiat?